notifications.yaml

```diff
@@ -0,0 +1,268 @@
+          An absolute URI that identifies the specific occurrence of the
+          problem.
+
+          It may or may not yield further information if dereferenced.
+  MessageContent:
+    title: MessageContent
+    type: object
+    properties:
+      body_short:
+        type: string
+        description: Message content (short version).
+        minLength: 3
+        maxLength: 100
+      body_long:
+        type: string
+        description: Message content (long version).
+        minLength: 3
+    required:
+      - body_short
+  MessageDefinition:
+    title: MessageDefinition
+    type: object
+    properties:
+      to:
+        description: The fiscal code of the destination user.
+        type: string
+      dry_run:
+        type: boolean
+        description: >-
+          This parameter, when set to true, allows developers to test a request
+          without actually sending a message.
+
+
+          The default value is false.
+      time_to_live:
+        type: integer
+        description: >-
+          This parameter specifies how long (in seconds) the message should be
+          kept in storage if the device is offline. The maximum time to live
+          supported is 1 year, and the default value is 1 hour.
+        minimum: 3600
+        maximum: 31536000
+      content:
+        $ref: '#/definitions/MessageContent'
+    required:
+      - to
+      - content
+  MessageStatus:
+    title: MessageStatus
+    type: object
+    properties:
+      status:
+        type: string
+        description: |
+          ACCEPTED: the message has been accepted for delivery
+
+          SENT: the message was delivered to the channel
+
+          FAILED: something wong happened, check the error code and message
+
+          TIMED_OUT:  in case a TTL was set
+        x-extensible-enum:
+          - ACCEPTED
+          - SENT
+          - TIMED_OUT
+          - FAILED
+      failed_at:
+        type: string
+        format: date-time
+      sent_at:
+        type: string
+        format: date-time
+      problem:
+        $ref: '#/definitions/ProblemJson'
+  AcceptedMessage:
+    title: AcceptedMessage
+    allOf:
+      - type: object
+        properties:
+          id:
+            type: string
+            format: uuid
+          accepted_at:
+            type: string
+            format: date-time
+          status:
+            type: object
+            default: {}
+            properties:
+              mail:
+                $ref: '#/definitions/MessageStatus'
+              sms:
+                $ref: '#/definitions/MessageStatus'
+              browser:
+                $ref: '#/definitions/MessageStatus'
+              push:
+                $ref: '#/definitions/MessageStatus'
+        required:
+          - id
+      - $ref: '#/definitions/MessageDefinition'
+  PaginationResponse:
+    type: object
+    title: pagination
+    description: Pagination response parameters (HATEOAS version).
+    properties:
+      index:
```

Comment:
it's an offset (taken straight from http://zalando.github.io/restful-api-guidelines/pagination/Pagination.html)